mta-multi-tenant.yaml

```diff
@@ -53,83 +64,107 @@ modules:
     path: app
     parameters:
       memory: 256M
       disk-quota: 512M
       keep-existing-routes: true
     properties:
-      TENANT_HOST_PATTERN: ^(.*)-${default-uri} # testing only, use custom domain with wildcard for production
+      TENANT_HOST_PATTERN: ^(.*)-${default-host}.${default-domain} # testing only, use custom domain with wildcard for production
     requires:
-    - name: srv-api
-      group: destinations
-      properties:
-        name: backend
-        url: ~{srv-url}
-        forwardAuthToken: true
-        strictSSL: true
-    - name: bookshop-mt-uaa
+      - name: srv-api
+        group: destinations
+        properties:
+          name: backend
+          url: ~{srv-cert-url}
+          forwardAuthCertificates: true
+          forwardAuthToken: true
+          strictSSL: true
+      - name: bookshop-identity
+        parameters:
+          config:
+            credential-type: "X509_GENERATED"
+            key-length: 2048
+            validity: 30
+            validity-type: "DAYS"
+            app-identifier: "microservice1"
+      - name: bookshop-sms
     provides:
       - name: app-api
         properties:
           app-url: '${default-url}'
-          app-domain: '${domain}'
+          app-domain: '${default-domain}'
 # --------------------- RESOURCES ---------------------
 resources:
-# -----------------------------------------------------
-  - name: bookshop-mt-uaa
-    type: org.cloudfoundry.managed-service
-    parameters:
-      service: xsuaa
-      service-plan: application
-      path: ./xs-security-mt.json
-      config: # override xsappname as it needs to be unique
-        xsappname: bookshop-mt-${org}-${space}
-        oauth2-configuration:
-          redirect-uris:
-          - https://*.~{app-api/app-domain}/**
-    requires:
-      - name: app-api
-  - name: bookshop-mt-service-manager
+  # -----------------------------------------------------
+  - name: bookshop-service-manager
     type: org.cloudfoundry.managed-service
     parameters:
       service: service-manager
       service-plan: container
-  - name: bookshop-mt-saas-registry
+  - name: bookshop-identity
     type: org.cloudfoundry.managed-service
     parameters:
-      service: saas-registry
+      service: identity
       service-plan: application
       config:
-        appName: bookshop-mt-${org}-${space} # this is the text on the tile
-        xsappname: bookshop-mt-${org}-${space} # this is the value from xsuaa.parameters.config.xsappname
-        appUrls:
-          getDependencies: ~{srv-api/srv-url}/mt/v1.0/subscriptions/dependencies
-          onSubscription: ~{srv-api/srv-url}/mt/v1.0/subscriptions/tenants/{tenantId}
-          onSubscriptionAsync: true
-          onUnSubscriptionAsync: true
-          onUpdateDependenciesAsync: true
-          callbackTimeoutMillis: 3600000
-        displayName: bookshop-java
-        description: A simple CAP Java project.
-        category: 'Category'
+        authorization:
```

Comment:
This is for AMS integration. Currently, we do not have one in the bookshop example. So, in principle it would not be required at the moment. We would require it if we add support for AMS-based authorizations which will happen probably soon.

Reply:
--> this afternoon ;-)